app/javascript/src/components/Dashboard/Notes/NoteTable.jsx

```diff
@@ -56,11 +73,37 @@ export default function NoteTable({
                 />
               </td>
               <td>
-                <div className="flex flex-row items-center justify-start text-gray-900">
+                <div className="flex flex-row items-center justify-start text-blue-700">
                   {note.title}
                 </div>
```

Comment:
Please use neetoUI Button with `style = "link"`. 

Please refer [neetoUI Button](https://neetoui.netlify.app/documentation#/button) for more details.